src/mlpack/bindings/go/print_model_util.hpp

```diff
@@ -262,68 +262,11 @@ void PrintModelUtilGo(
  */
 template<typename T>
 void PrintModelUtilGo(
-    const util::ParamData& d,
+    const util::ParamData& /* d */,
     const typename boost::disable_if<arma::is_arma_type<T>>::type* = 0,
     const typename boost::enable_if<data::HasSerialize<T>>::type* = 0)
 {
-  // First, we have to parse the type.  If we have something like, e.g.,
-  // 'LogisticRegression<>', we must convert this to 'LogisticRegression[].'
-  std::string goStrippedType, strippedType, printedType, defaultsType;
-  StripType(d.cppType, goStrippedType, strippedType, printedType, defaultsType);
-
-  /**
-   * This gives us code like:
-   *
-   *  type \<Type\> struct {
-   *    mem unsafe.Pointer
-   *  }
-   *
-   */
-  std::cout << "type " << goStrippedType << " struct {" << std::endl;
-  std::cout << "  mem unsafe.Pointer" << std::endl;
-  std::cout << "}" << std::endl;
-  std::cout << std::endl;
-
-  /**
-   * This gives us code like:
-   *
-   *  func (m *\<Type\>) alloc\<Type\>(identifier string) {
-   *    m.mem = C.mlpackGet\<Type\>Ptr(C.CString(identifier))
-   *    runtime.KeepAlive(m)
-   *  }
-   *
-   */
-  std::cout << "func (m *" << goStrippedType << ") alloc"
-            << strippedType << "(identifier string) {" << std::endl;
-  std::cout << "  m.mem = C.mlpackGet" << strippedType
-            << "Ptr(C.CString(identifier))" << std::endl;
-  std::cout << "  runtime.KeepAlive(m)" << std::endl;
-  std::cout << "}" << std::endl;
-  std::cout << std::endl;
-
-  /**
-   * This gives us code like:
-   *
-   *  func (m *\<Type\>) get\<Type\>(identifier string) {
-   *    m.alloc\<Type\>(identifier)
-   *  }
-   *
-   */
-  std::cout << "func (m *" << goStrippedType << ") get"
-            << strippedType << "(identifier string) {" << std::endl;
-  std::cout << "  m.alloc" << strippedType << "(identifier)" << std::endl;
-  std::cout << "}" << std::endl;
-  std::cout << std::endl;
-
-  // Print function to set specified mlpack parameter object ptr from Go.
-  std::cout << "func set" << strippedType
-            << "(identifier string, ptr *" << goStrippedType << ") {"
-            << std::endl;
-  std::cout << "  C.mlpackSet" << strippedType
-            << "Ptr(C.CString(identifier), (unsafe.Pointer)(ptr.mem))"
-            << std::endl;
-  std::cout << "}" << std::endl;
-  std::cout << std::endl;
+  // Do nothing.
```

Comment:
I think we can probably remove this entire file now---or did I overlook something?

Reply:
Thank you for pointing this out. Previously this file was also used for generating .h/.cpp files, But now I used similar approach used by julia for generating .h/.cpp files. This approach will also save some time for build the go-bindings .